chiaswap/main.py

```diff
@@ -619,7 +626,7 @@ def have_btc_want_xch(logfile, secret_key, btc_amount, xch_amount_mojos):
 
     while True:
         print(
-            "Enter your counterparty private key OR the lightning invoice receipt pre-image or `quit`"
+            "Enter your counter-party private key OR the lightning invoice receipt pre-image or `quit`"
```

Comment:
A cursory Google search suggests that "counterparty" is a word and that it makes sense here.